apps/examples/testcase/le_tc/network/tc_net_recv.c

```diff
@@ -159,7 +159,7 @@ void *recv_server(void *args)
 	struct sockaddr_in sa;
 	int SocketFD = socket(PF_INET, SOCK_STREAM, IPPROTO_TCP);
 
-	if(SocketFD < 0) {
+	if (SocketFD < 0) {
```

Comment:
[M12] No mixed-case, Use underscores('_') to separate words in an name.
ref : https://wiki.tizen.org/Native_Platform_Coding_Idiom_and_Style_Guide

Reply:
@KimDongEon , same coding rule applies to `ConnectFD` also right!!